app/Models/Achievement.php

```diff
@@ -176,13 +176,14 @@ public function getActivitylogOptions(): LogOptions
     {
         return LogOptions::defaults()
             ->logOnly([
-                'Title',
+                'AssocVideo',
```

Comment:
Added `AssocVideo` here. We don't currently have an edit field on the achievement resource for it though.